### DISCUSSION TITLE
Glossary of cFS/cFE acronyms

### DISCUSSION THREAD

**Discussion by ghost** [nasa/cFS#738] [Q&A] [answered]
Hi ~ 

Is there a more in-depth raw text lookup (or even a pdf collection) of cFS/cFE acronyms? I see this [here based on your cFE App Developers User Guide](https://github.com/nasa/cFE/blob/main/docs/cFE%20Application%20Developers%20Guide.md#15-glossary-of-terms). I'm reading through these [power point training slides](https://ntrs.nasa.gov/api/citations/20210022378/downloads/NASA%20TM%2020205000691%20REV%202%20Final.pdf) and see a few abbreviations I just don't know what they are. Maybe even a overall NASAS glossary would be helpful to.

**Comment by thnkslprpt** [comment, marked as answer]
Try this search engine: https://nasaacronyms.com/

By the way, there is a more recent version of the training slides available now: https://ntrs.nasa.gov/api/citations/20240000217/downloads/cFS_Training_Draco.pptx
